cmake/modules/Buildpmem.cmake

```diff
@@ -25,11 +25,7 @@ function(build_pmem)
   ExternalProject_Add(pmdk_ext
       ${source_dir_args}
       CONFIGURE_COMMAND ""
-      # Explicitly built w/o NDCTL, otherwise if ndtcl is present on the
-      # build system tests statically linking to librbd (which uses
-      # libpmemobj) will not link (because we don't build the ndctl
-      # static library here).
-      BUILD_COMMAND ${make_cmd} CC=${CMAKE_C_COMPILER} NDCTL_ENABLE=n BUILD_EXAMPLES=n BUILD_BENCHMARKS=n DOC=n
+      BUILD_COMMAND ${make_cmd} CC=${CMAKE_C_COMPILER} BUILD_EXAMPLES=n BUILD_BENCHMARKS=n DOC=n
```

Comment:
@optimistyzy hi Ziye, could you help review this change?

Reply:
This removes "Not building NDCTL_ENABLE". I think this change is OK to me.